content/en/docs/tutorials/kubernetes-basics/expose/expose-intro.html

```diff
@@ -94,7 +94,7 @@ <h3>Services and Labels</h3>
 		</div>
 		<div class="row">
 			<div class="col-md-12">
-				<h3>Create a new Service</h3>
+				<h3>Step 1: Create a new Service</h3>
```

Comment:
```suggestion
				<h3>Step 1: Creating a new Service</h3>
```
Should we use a consistent expression?

Reply:
Yes, that looks great. The documentation page maintains a consistent appearance throughout.Thanks :)